gammapy/obs/observation.py

```diff
@@ -38,24 +41,28 @@ class ObservationTable(Table):
     """Observation table (a.k.a. run list).
 
     This is an `~astropy.table.Table` sub-class, with a few
-    convenience methods and the following columns:
-
-    * ``OBS_ID``
-    * ``ONTIME``
-    * ``LIVETIME``
-    * ...
-
+    convenience methods. The format of the observation table
+    is described in :ref:`dataformats_observation_lists`.
     """
 
     def info(self):
         ss = 'Observation table:\n'
+        obs_name = self.meta['OBSERVATORY_NAME']
+        ss += 'Observatory name: {}\n'.format(obs_name)
         ss += 'Number of observations: {}\n'.format(len(self))
-        ontime = self['ONTIME'].sum()
+        ontime = Quantity(self['TIME_OBSERVATION'].sum(), self['TIME_OBSERVATION'].unit)
         ss += 'Total observation time: {}\n'.format(ontime)
-        livetime = self['LIVETIME'].sum()
+        livetime = Quantity(self['TIME_LIVE'].sum(), self['TIME_LIVE'].unit)
         ss += 'Total live time: {}\n'.format(livetime)
         dtf = 100. * (1 - livetime / ontime)
-        ss += 'Average dead time fraction: {:5.2f}%'.format(dtf)
+        ss += 'Average dead time fraction: {:5.2f}%\n'.format(dtf)
+        time_ref = time_ref_from_dict(self.meta)
+        time_ref_unit = time_ref_from_dict(self.meta).format
+        ss += 'Time reference: {} {}'.format(time_ref, time_ref_unit)
+        # TODO: for an unknown reason, I can't enable astropy.units.cds
```

Comment:
Let's try to reduce the number of new TODOs in Gammapy. The options are:
* You try to figure it out and start using it.
* Or, if it doesn't work, comment here what you've tried and want to achieve and I'll investigate.
* You let it go because we don't really need this at all and remove the TODO.